packages/app/templates/extensions/projects/flow_trigger/shopify.extension.toml.liquid

```diff
@@ -1,18 +1,12 @@
-name = "{{name}}"
+name = "{{ name }}"
+description = "Your description"
 type = "flow_trigger"
 
```

Comment:
as mentioned before, you need an extensions array, otherwise this won't be compatible with the future unified config. Even if the extension object in the array only contains the `type`, it should be mandatory.